device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/https/HttpsIotHubConnection.java

```diff
@@ -151,11 +151,6 @@ public IotHubStatusCode sendMessage(Message message) throws TransportException
                 //Codes_SRS_HTTPSIOTHUBCONNECTION_34_067: [If the response from the service is OK or OK_EMPTY, this function shall notify its listener that a message was sent with no exception.]
                 this.listener.onMessageSent(transportMessage, null);
             }
```

Comment:
Can you point to where it is handled?

Reply:
Never mind. Should have read description earlier :)

Reply:
Sure, I can add a note that non-200 codes have their exceptions handled in the transport layer

Reply:
That would be good